kit/Kit.cpp

```diff
@@ -1346,6 +1348,17 @@ class Document final : public DocumentManagerInterface
         return _docPasswordType;
     }
 
+    void updateActivityHeader() const override
+    {
+        // pre-prepare and set details in case of a signal later
+        std::stringstream ss;
+        ss << "Session count: " << _sessions.size() << "\n";
+        for (const auto& it : _sessions)
+            ss << "\t" << it.second->getActivityState() << "\n";
```

Comment:
We have some places where we, like here, just deref "it.second" but then we have some other places like drainQueue and findSessionByViewId kit/Kit.cpp where we null check it.second.

Seems that the null check ones are just excessively cautious and it.second is always dereferenceable.